### DISCUSSION TITLE
Forager's code of ethics

### DISCUSSION THREAD

**Discussion by ezwelty** [falling-fruit/falling-fruit-web#931] [Ideas]
For the ethics of urban foraging, we point to Urban Edibles' ethics page (http://urbanedibles.org/project/ethics), but that page is now down. Anyway, I think it's time for us to put together our own code of ethics, as we are now an influential player in the foraging world. It should be very concise and use simple language so that it can be translated easily.

As a starting point, here is the content from the now-defunct Urban Edibles page:

### Ethical Considerations

This project cannot exist unless every participant acts responsibly. Be mindful of the sources you pick from and submit to the database. Here is a short working list of considerations.

### Asking Permission

Always ask permission before you pick no matter where the trees or bushes are located. Not everyone is alright with their neighbors picking from their sidewalk plot. However, many fruit-producing trees and herbaceous species are readily available on public land. It’s not always easy to tell which is which. Part of being a thoughtful community member is in respecting both private and public terrain. The Urban Edibles wild food source database is designed as a resource for potential harvesting locations (i.e. areas with an abundance of fallen fruit). We do not condone unsanctioned harvesting practices or trespassing. Consistently asking permission to harvest wild foods ensures lawful conduct. It also promotes face-to-face dialog between you and your neighbors. We believe that building this kind of wild food network helps connect us to one another as well as our urban habitat. It makes our communities that much healthier!

### How much do I really need?

A tree full of ripe black cherries can be really exciting but how many will you use before they go bad? How much can you carry? Decide before you pick. Over-harvesting a wild food source can be very counter-intuitive when the goods go bad.

### Will my harvesting leave an impact?

This includes visual impact, impact for future harvesters and last but not least the impact on the particular plant you are picking from. It is imperative to pick in a balanced and selective manner. The last thing we want is to damage the sources from which we harvest! You can help a public source produce better next year by watering it or notifying the city if it needs pruning.

### Consider chemical contamination

Watch out! It’s easy to forget that Portland is a major metropolitan area with a strong potential for toxicity. Engaging dialog with homeowners and park workers about the history of a site is always helpful. Paint chips, pesticides, motor oil spills and even car wash runoff can affect the quality of the sources you pick from. The Portland Parks website claims that “the vast majority of pest management practices in parks never involve the use of pesticides” (Source). However, this is not always so. A vast majority of homeowners also tend to implement domestic pesticides like Roundup as well. Needless to say, it’s always best to glean as much information from the harvesting site as you can.

### Lawful Considerations

Legalities related to harvesting wild food sources are not always clear. However, the City of Portland has several laws and regulations pertaining to urban forestry practices such as Chapter 20.40 of the Portland City Code and Charter.

**Comment by niccokunzmann** [comment]
See also:
- https://mundraub.org/mundraeuber-regeln
- https://na-ovoce.cz/web/kodex
- https://github.com/niccokunzmann/mundraub-android/blob/cc59722d3263cc25048146d6a911ddf11a122e65/app/src/main/res/values/strings.xml#L494

**Comment by BrittanyBunk** [comment]
there's some thoughts about mixing in rewilding in with 'leaving an impact'. Instead of just being a minimalist and taking, why not create abundance and give too? Encourage more harvesting by planting more in ravaged lands? I get that this might be part of other projects, but I haven't seen any rewilding maps. Maybe if there is rewilding initiatives - then they can be labeled that - so people take extra caution not to eat someone else's work?

**Reply by aokegbile** [reply]
I think rewilding is interesting, would this be a feature added to falling fruit, or a separate project entirely?

**Reply by ezwelty** [reply]
I don't think there's capacity to develop dedicated features for related but separate initiatives, but maybe existing features can be used. For example, we have a flexible location tagging system which allows a freegan / dumpster diving community to thrive on Falling Fruit. What types of locations would one want to add to or find on a rewilding map – "vacant land"? We can also make sure to mention guerrilla gardening, rewilding, needs of non-human animals, etc in a Falling Fruit code/manifesto.

**Comment by ezwelty** [comment]
I've compiled several of the foraging guidelines together here as a starting point for pulling together an official one for Falling Fruit:
https://docs.google.com/document/d/1MBUptjCVOCXlsPdtfbpzDGWYKwO5am2QDkm_89PrEKA
